programmers/다트.js

```diff
@@ -0,0 +1,40 @@
+function solution(dart) {
+    
+    let num = 0;
+    let answer = [];
+    let temp = 0;
+    
+    for(let i = 0; i < dart.length; i++) {
+        if(dart[i] >= 0 && dart[i] <= 9) {
+            if(dart[i] == 1 && dart[i+1] == 0) {        // 10점일 때
+                temp = 10;
+                i++;
+            }
+            else {
+                temp = dart[i];
+            }
+        }
+        if(dart[i] === 'S') {
+            answer.push(temp);
+        }
+        if(dart[i] === 'D') {
+            answer.push(temp ** 2);
```

Comment:
```suggestion
            answer.push(Math.pow(temp, 2));
```
Math.pow 메서드는 제곱근 메서드인데 이런 방법도 있다~ 라고 알아두시면 좋을것같아요